clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -75,6 +82,8 @@
 
     /* max time in ms for the producer to wait for acknowledgement from server*/
     private final int requestTimeoutMs;
+    
+    private final Time time;
```

Comment:
@enothereska @guozhangwang Sounds good. If we're planning to implement Ewen's proposed Now solution, then the local Time reference will go away anyway.